Scripts/Resource Count/test.yaml

```diff
@@ -0,0 +1,4 @@
+scripts: 
+  - resource-count.yolol
+cases: []

```

Comment:
Could you add some test cases? These help:
 - Assure that the script works as expected
 - Help explain what's going on to someone trying to understand the script
 - Check that future modifications to the script don't break things.

Thanks :)

documentation on how testing works: https://dbaumgarten.github.io/yodk/#/cli?id=testing

Reply:
Since you know how long the script is you probably want to set `maxlines: 7` - this means every test case will run 12 lines of the program (down to the goto) and terminate. Or if the script holds some state from one run to the next maybe set it a bit higher (e.g. `maxlines: 28` to run the entire program 4 times).